### DISCUSSION TITLE
Offline trace collection

### DISCUSSION THREAD

**Discussion by sirzooro** [jaegertracing/jaeger#3543] [Q&A]
Hi,
We would like to use OpenTelemetry with Jaeger in our product. However we have one major issue: because of CPU and RAM limitations we cannot deploy ElasticSearch or Cassandra on target system, and usually there is no Internet access there too. Fortunately disk space is not so limited there, so we could store all collected data in a file, and later import it to Jaeger on another machine with more CPU and RAM.

At this moment we think about implementing custom OpenTelemetry exporter to write data to file, and tool which would load this file and "replay" it via OpenTelemetry API to Jaeger. Is there a better way to do this? Most likely we would be able to deploy Jaeger Agent and Collector on target system.

**Comment by yurishkuro** [comment]
You should skip the agent, there's no need for it in this setup.

**Comment by sirzooro** [comment]
Yes, for solution proposed in initial post above they are not needed. I am looking for alternative solutions which may need agent. collector and possibly some other piece of software, which is not very CPU- and RAM-heavy.
